web/cobrands/fixmystreet/assets.js

```diff
@@ -24,6 +24,8 @@ OpenLayers.Layer.VectorBase = OpenLayers.Class(OpenLayers.Layer.Vector, {
         $(fixmystreet).on('assets:unselected', this.checkSelected.bind(this));
         $(fixmystreet).on('report_new:category_change', this.update_layer_visibility.bind(this));
         $(fixmystreet).on('inspect_form:asset_change', this.update_layer_visibility.bind(this));
+        $(fixmystreet).on('maps:update_pin', this.checkSelected.bind(this));
+        $(fixmystreet).on('report_new:category_change', this.checkSelected.bind(this));
```

Comment:
Whilst there might well be an argument that we should do this, I think for a commit that's just moving like this we should copy Bucks/Peterborough/TfL and create a NorthamptonshireVectorAsset class that adds these two event handlers. I'm assuming there is a reason why they're not in the Base, though perhaps there isn't, but it probably needs more checking than just for this.

Reply:
Yeah I was a bit unsure about this, but was needed to get the Cypress tests passing.

I'll move it into a `NorthamptonshireVectorAsset` as you suggest for now.